### PR TITLE
OP(fake_quantize) error message enhancement

### PR DESCRIPTION
**Pull request by wzzju**
## fake_quantize op报错信息不合规增强：

1. 将fake_quantize一类Op中的InferShape方法中的输入输出检查从 `PADDLE_ENFORCE `改为`OP_INOUT_CHECK `。

2. 将对属性`bit_length`对检查由`PADDLE_ENFORCE`改为 `PADDLE_ENFORCE_EQ`，并细化说明了设置的不规范值为多少。

### CODE REVIEW

File: paddle/fluid/operators/fake_quantize_op.cc [hunk 1]
```diff
@@ -211,8 +210,9 @@ class FakeQuantizeAbsMaxOpMaker : public framework::OpProtoAndCheckerMaker {
     AddAttr<int>("bit_length", "(int, default 8)")
         .SetDefault(8)
         .AddCustomChecker([](const int& bit_length) {
-          PADDLE_ENFORCE(bit_length >= 1 && bit_length <= 16,
-                         "'bit_length' should be between 1 and 16.");
+          PADDLE_ENFORCE_EQ(bit_length >= 1 && bit_length <= 16, true,
+                            platform::errors::InvalidArgument(
+                                "'bit_length' should be between 1 and 16."));
```
Comment (chenwhql) [inline paddle/fluid/operators/fake_quantize_op.cc]: add `but received is %d`?

File: paddle/fluid/operators/fake_quantize_op.cc [hunk 2]
```diff
@@ -263,8 +261,9 @@ class FakeChannelWiseQuantizeAbsMaxOpMaker
...
```
Comment (chenwhql) [inline paddle/fluid/operators/fake_quantize_op.cc]: same as above

File: paddle/fluid/operators/fake_quantize_op.cc [hunk 3]
```diff
@@ -329,8 +325,9 @@ class FakeQuantizeRangeAbsMaxOpMaker
     AddAttr<int>("bit_length", "(int, default 8), quantization bit number.")
         .SetDefault(8)
         .AddCustomChecker([](const int& bit_length) {
-          PADDLE_ENFORCE(bit_length >= 1 && bit_length <= 16,
-                         "'bit_length' should be between 1 and 16.");
+          PADDLE_ENFORCE_EQ(bit_length >= 1 && bit_length <= 16, true,
+                            platform::errors::InvalidArgument(
+                                "'bit_length' should be between 1 and 16."));
```
Comment (chenwhql) [inline paddle/fluid/operators/fake_quantize_op.cc]: same as above

File: paddle/fluid/operators/fake_quantize_op.cc [hunk 4]
```diff
@@ -404,8 +397,9 @@ class FakeQuantOrWithDequantMovingAverageAbsMaxOpMaker
...
```
Comment (chenwhql) [inline paddle/fluid/operators/fake_quantize_op.cc]: same as above

File: paddle/fluid/operators/fake_quantize_op.cc (hunk 1 above)
Comment (wzzju) [reply paddle/fluid/operators/fake_quantize_op.cc]: Done.

File: paddle/fluid/operators/fake_quantize_op.cc (hunk 2 above)
Comment (wzzju) [reply paddle/fluid/operators/fake_quantize_op.cc]: Done.

File: paddle/fluid/operators/fake_quantize_op.cc (hunk 4 above)
Comment (wzzju) [reply paddle/fluid/operators/fake_quantize_op.cc]: Done.

**Review by sneaxiy — APPROVED** [review]

**Pull request merged**